schemas/preferences.sublime-settings.json

```diff
@@ -1262,6 +1262,21 @@
           "markdownDescription": "The maximum file size (in bytes) after which syntax highlighting will be disabled for files and will be shown as plain text.",
           "type": "integer",
           "default": 16777216
+        },
+        "find_scroll_highlights_limit": {
+          "markdownDescription": "The maximum number of find results to show in the scroll bar. If this number is exceeded no results will be shown. Use `0` to remove this limit.",
+          "type": "integer",
+          "default": 8192
+        },
+        "find_highlight_matches_max_size": {
+          "markdownDescription": "The maximum file size (in bytes) for which the find option **Highlight matches** will highlight matches (affects non-regexp searches). Use `0` to remove this limit.",
+          "type": "integer",
+          "default": 16777216
+        },
+        "find_regex_highlight_matches_max_size": {
```

Comment:
I believe all of those should be within the `EditorSettings` object as those appear to be supported in syntax-specific or project settings.

Reply:
I don't think I understand ? What `EditorSettings` object ?

Reply:
Those settings should be inserted within this `properties` object: https://github.com/sublimelsp/LSP-json/blob/abfb435901e8cf4c2b1ec988842482f05287bdf4/schemas/preferences.sublime-settings.json#L9-L10

Reply:
Now that I look at this `preferences.sublime-settings.json` more closely, it seems to be divided into `EditorSettings` and `allOf`. What's the difference ?

Reply:
The preferences within `EditorSettings` are also supported in syntax-specific and project settings while all other are not.

Reply:
Since I was pinged, I'll point out that this is still not addressed.

Reply:
Yes. I am aware that I have not addressed this. I had a busy last week. I will get this addressed this weekend. There appears to be a couple more settings added in 4114 that also need to go along with this PR that I missed adding.

Reply:
I have addressed this.